### PR TITLE
container-bridge mode with UCR

### PR DESCRIPTION
**Pull request by sascala**
https://jira.mesosphere.com/browse/MARATHON-7186

I am very happy to consider other appropriate places in the Marathon docs for this information.

### CODE REVIEW

File: docs/docs/networking.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ VIPs map traffic from a single virtual address to multiple IP addresses and port
 Several [command-line flags](command-line-flags.html) determine Marathon's behavior with respect to networking.
 
 * `default_network_name` is injected as the `name` of a `container` mode network when left blank by an application.
-* `local_port_min` and `local_port_max` define a port range from which Marathon automatically allocates *service-port*s.
+* `local_port_min` and `local_port_max` define a port range from which Marathon automatically allocates \*service-port\*s.
```
Comment (jdef) [inline docs/docs/networking.md]: curious, why the backslashes here?

File: docs/docs/networking.md [hunk 2]
```diff
@@ -298,6 +298,47 @@ Specify `container/bridge` mode through the `networks` property:
...
+##### Installing the `mesos-bridge` CNI plugin
+If you are not using DC/OS and want to enable `container/bridge` mode with the Universal Container Runtime (UCR), several more steps are necessary to install and use the `mesos-bridge` CNI plugin.
+
+**Prerequisites**
+- Mesos version 1.2.0 or higher.
+- Marathon version 1.5 or higher.
+
+1. Navigate to or create a `/var/lib/mesos/cni` directory on each of your agent nodes as well as `config` and `plugins` subdirectories.
+
+1. Clone the CNI repository from [https://github.com/containernetworking/cni](https://github.com/containernetworking/cni) and build according to their instructions.
```
Comment (jdef) [inline docs/docs/networking.md]: I'd make this clone-and-build the first step:

1. It's likely the step that will take the longest.
2. The remaining steps become "create this file/dir" and "copy these things there", logically grouped together
3. Currently, it's not super clear if I'm supposed to clone the repo into the directory structure I just created.

File: docs/docs/networking.md (hunk 1 above)
Comment (sascala) [reply docs/docs/networking.md]: if you don't escape the asterisks, the whole text looks like it's in italics in the text editor.

File: docs/docs/networking.md (hunk 2 above)
Comment (sascala) [reply docs/docs/networking.md]: can do.

**Comment by sascala** [comment]
@jdef -- comments addressed

**Pull request merged**